ui/lib/ldap/addon/components/page/roles.hbs

```diff
@@ -0,0 +1,98 @@
+<TabPageHeader @model={{@backendModel}} @breadcrumbs={{@breadcrumbs}}>
+  <:toolbarFilters>
+    {{#if (and (not @promptConfig) @roles)}}
+      <div class="field">
+        <p class="control has-icons-left">
+          <Input class="filter input" placeholder="Filter roles" data-test-roles-filter {{on "input" this.onFilterInput}} />
+          <Icon @name="search" class="search-icon has-text-grey-light" />
+        </p>
+      </div>
```

Comment:
I wanted to break away from the pattern of using `NavigateInput` since it is overkill and a transition is not necessary in this case since we already have all of the data loaded. Instead, the roles passed down from the route are simply filtered in the component.

Reply:
I kicked around the idea but thought that since it's just an input with an input event there might not be much of an advantage of having a component. I suppose that the input event is debounced so it would cut down on repeating that. I could make a `DebouncedInput` component in the core addon.